src/builders/dev-server/index.ts

```diff
@@ -24,6 +26,17 @@ export class SingleSpaDevServer extends DevServerBuilder {
         SINGLE-SPA-ANGULAR: Angular dev server is serving application as a single module`);
         return webpackConfig;
     }    
+    _buildServerConfig(root, projectRoot, options, browserOptions) {
+        // @ts-ignore
+        const devServerConfig = super._buildServerConfig(root, projectRoot, options, browserOptions);
+
+        return webpackMerge.smart(devServerConfig, {
+            // @ts-ignore
```

Comment:
the typescript definition that is used by webpack-merge doesn't know that `contentBase` is a thing, but [it is](https://webpack.js.org/configuration/dev-server/#devservercontentbase)